src/utils.ts

```diff
@@ -341,3 +341,21 @@ export function getFormattedDateTime(): string {
 
   return `${date}-${time}`;
 }
+
+export const getStringSizeInMb = (str: string): number => {
```

Comment:
i am assuming this for handling unicode. is there any library for this??

Reply:
yes, this calculates the size of the history object by handling unicode characters